### PR TITLE
fix: remove noindex tag & proxy sitemaps

### PR DESCRIPTION
**Pull request by alexey-yarmosh**
Fixes https://github.com/jsdelivr/www.jsdelivr.com/issues/542

Note. In this PR `harmon` breaks sitemap.xsl file, that need to be fixed in future:

image

### CODE REVIEW

**Comment by MartinKolarik** [comment]
Causes some JS issues:
image

File: src/proxy.js [hunk 1]
```diff
@@ -161,7 +170,7 @@ module.exports = (proxyTarget, host) => {
 						});
 				},
 			};
-		})), true),
+		})), false),
```
Comment (MartinKolarik) [inline src/proxy.js:173]: We certainly can't run this on everything, only HTML + XML.

Comment (alexey-yarmosh) [reply src/proxy.js:173]: true, I'll take it

Comment (MartinKolarik) [reply src/proxy.js:173]: 👍 we can probably keep it at `true` and do our own filtering based on response headers if there's no suitable option in the package directly

Comment (alexey-yarmosh) [reply src/proxy.js:173]: Here is the PR with a slightly another solution please check if it suits us. Seems to work as expected.
https://github.com/jsdelivr/www.jsdelivr.com/pull/553

**Pull request merged**